tests/integration/test_secure_message.py

```diff
@@ -5,7 +5,7 @@
 
 from frontstage import app
 from frontstage.exceptions.exceptions import IncorrectAccountAccessError
-from tests.integration.mocked_services import (conversation_json, conversation_list_json,
```

Comment:
Why have we renamed conversation to message? I thought message was singular and conversation was multiple messages?

Reply:
It's because the conversation_json was a misnomer.  The contents of `conversation_json` has just a single message in a conversation.  A lot of the things that use it tend to wrap it in a list (as a conversation is a list of messages) so it's clearer what is it with this name change

Reply:
ah ok that's fair though then